### DISCUSSION TITLE
How to keep duplicate columns from showing in Time in Board Column view?

### DISCUSSION THREAD

**Discussion by rrobertsor** [nbrown02/FlowViz#104] [General]
I'm struggling with being able to remove duplicate column references and each has some data tied to it. 

How do I get one visual on "waiting for other teams" vs 3 for a single work item?

Thanks!
~Rebecc
![image](https://github.com/nbrown02/FlowViz/assets/145143470/cd67bfc9-7472-4b6f-95dc-849a7699d063)
a

**Comment by nbrown02** [comment]
Hey Rebecca - thanks for sharing this :)

Duplicate columns can occur when, in the given time period, there may have been board columns added to the board so that the board 'column order' has multiple entries.

For example if I had a board of:

_To Do | Doing | Done_

They have a column order 1 | 2 | 3

If an item is in Doing and then you add a column called Refining so your board is now:

_To Do | Refining | Doing | Done_

Then 'Doing' now has a column order of 3 (and thus appears twice on the visual).

Similarly, if a work item appears in multiple boards (for example shared area paths) this can also occur.

Unfortunately it's difficult for me to test for scenarios like this (hence feedback is great) - I will see if anything can be done to prevent it...

**Reply by rrobertsor** [reply]
Okay, thank you! I appreciate your help. So technically, if we add the sum of those columns together that = total time in that column? Or do I have to be concerned about the column names not = the same name?

**Reply by nbrown02** [reply]
Yup if they have the same name that's the total time in that column. If it's different names then it really depends on how different they are :)

**Reply by rrobertsor** [reply]
Thanks for your help!
One more question: I was able to export the data from that visual and pull into excel to combine those rows where there were duplicates as a workaround for now. Can you advise, how do you come to the # of days in that column? Anyway to walk me through what goes into that Column 2 table as I only see that and the blocked tables connected in that Time in Board Column view. So I am not sure how  you were able to get to # of days.

**Reply by nbrown02** [reply]
The time not blocked comes from the 'Not Blocked' column in the WorkItems Flow Efficiency table - which is just a SUM:

![image](https://github.com/nbrown02/FlowViz/assets/29369962/75035297-4477-4ddc-a0f7-1b98892a5c2f)

**Reply by nbrown02** [reply]
@rrobertsor could you try [download the latest version](https://github.com/nbrown02/FlowViz/raw/main/FlowViz%20(Project).pbit) and see if that fixes this issue? I think I've found a way to address it :)
